sql/errors.go

```diff
@@ -388,6 +388,12 @@ var (
 
 	// ErrUserCreationFailure is returned when attempting to create a user and it fails for any reason.
 	ErrUserCreationFailure = errors.NewKind("Operation CREATE USER failed for %s")
+
+	// ErrPrivilegeCheckFailed is returned when a user does not have the correct privileges to perform an operation.
+	ErrPrivilegeCheckFailed = errors.NewKind("%s command denied to user %s for table '%s'")
+
+	// ErrGrantUserDoesNotExist is returned when a user does not exist when attempting to grant them privileges.
+	ErrGrantUserDoesNotExist = errors.NewKind("You are not allowed to create a user with GRANT")
```

Comment:
This name doesn't seem to match the text

Reply:
That's intentional, this is the exact error returned by MySQL, but the variable name is the situation that it's returned in. You'll probably see this for quite a few errors I add.